src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier supplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
```

Comment:
Consider using a constant for the size of the figures array, like `private static final int ARRAY_SIZE = 6;`. This makes the code more readable and easy to maintain.